core/lib/config/src/configs/object_store.rs

```diff
@@ -45,28 +46,52 @@ impl ObjectStoreConfig {
 #[config(tag = "mode")]
 #[serde(tag = "mode")]
 pub enum ObjectStoreMode {
+    /// Writable GCS bucket with ambient authentication.
     #[config(alias = "Gcs")]
-    GCS { bucket_base_url: String },
+    GCS {
+        /// Name of the bucket.
+        // FIXME: this isn't a URL; would it make sense to rename it to `bucket_name`?
+        bucket_base_url: String,
```

Comment:
Subj. IMO, such renaming would make sense, and the old name could be retained as an (deprecated?) alias.